subschemas/poap-credential/src/resolvers/index.ts

```diff
@@ -0,0 +1,5 @@
+import Query from './Query';
```

Comment:
maybe just remove the query folder and make it's index file `Query.ts`, the existing structure is for when there are a lot of queries